### PR TITLE
 ownprops - teams

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

File: shared/teams/role-picker/controlled-container.js [hunk 1]
```diff
@@ -23,8 +23,16 @@ export type ControlledRolePickerProps = {
   showNotificationCheckbox?: boolean,
   sendNotificationChecked?: boolean,
   styleCover?: Object,
+  teamname: string,
 }
 
+type OwnProps = RouteProps<
+  {
+    ...ControlledRolePickerProps,
```
Comment (MarcoPolo) [inline shared/teams/role-picker/controlled-container.js]: Why like this instead of just `ControlledRolePickerProps` ?

Comment (chrisnojima) [reply shared/teams/role-picker/controlled-container.js]: i wasn't sure if more stuff would have to be hoisted in there or not while doing it

File: shared/teams/invite-by-email/container.native.js [hunk 2]
```diff
@@ -121,7 +124,7 @@ export default compose(
         _setHasPermission: () => hasPermission => ({hasPermission}),
       }
     ),
-    withPropsOnChange(['onExitSearch'], props => ({
```
[the remark this answers is not in the document]
Comment (chrisnojima) [reply shared/teams/invite-by-email/container.native.js]: i think this gets called once always, but it likely never mutated?

**Pull request merged**